pyblish_lite/model.py

```diff
@@ -629,16 +630,17 @@ def filterAcceptsRow(self, source_row, source_parent):
         """Exclude items in `self.excludes`"""
         model = self.sourceModel()
         item = model.items[source_row]
-
         key = getattr(item, "filter", None)
         if key is not None:
             regex = self.filterRegExp()
             if regex.pattern():
                 match = regex.indexIn(key)
                 return False if match == -1 else True
-
         for role, values in self.includes.items():
             data = getattr(item, role, None)
+            if hasattr(data, '__iter__'):
+                return True if any(value in values for value in data) else False
```

Comment:
I've incremented the path version and included all of the mentioned changes in:
[6319b981825a4d27af164d504b753ac2b543ef1f](https://github.com/p4vv37/pyblish-lite/commit/6319b981825a4d27af164d504b753ac2b543ef1f
)
As about reproducible, I'm not sure if I understand correctly what you mean, but as about screenshots, could You tell me where should I pu them

Regards